tutor/templates/apps/openedx/settings/partials/common_lms.py

```diff
@@ -37,6 +37,10 @@
     "LOCATION": "staticfiles_lms",
 }
 
+# Enable search features
+FEATURES["ENABLE_COURSE_DISCOVERY"] = True
```

Comment:
Why is course-discovery enabled by default in here?

Reply:
Agreed -- the course search features need these feature flags, not `ENABLE_COURSE_DISCOVERY` ([ref](https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/edx_search.html#lms)):
* `ENABLE_COURSEWARE_SEARCH`
* `ENABLE_DASHBOARD_SEARCH`